libtextsecure/errors.js

```diff
@@ -127,18 +127,14 @@
   }
   inherit(Error, UnregisteredUserError);
 
-  function EmptySwarmError(number, error) {
+  function EmptySwarmError(number, message) {
     // eslint-disable-next-line prefer-destructuring
     this.number = number.split('.')[0];
 
     ReplayableError.call(this, {
       name: 'EmptySwarmError',
-      message: 'Could not get any swarm nodes to query',
+      message,
     });
-
-    if (error) {
-      appendStack(this, error);
```

Comment:
Any idea what this line was for?
Do we not call this anymore then?

Reply:
Could probably change this to take an error and a message, the problem was that currently there wouldn't be any information about *where* the empty swarm error was occurring (i.e. we couldn't get any to begin with or we couldn't connect to any of the nodes we got in our list)
The error was usually just like a timeout or connection error, could still be useful tho I guess

Reply:
I take your word for it that we can afford to ditch that :)